symengine/mp_class.h

```diff
@@ -571,7 +571,7 @@ inline void mp_pow_ui(fmpz_wrapper &res, const fmpz_wrapper &i, unsigned long n)
     fmpz_pow_ui(res.get_fmpz_t(), i.get_fmpz_t(), n);
 }
 
-inline void mp_pow_ui(fmpq_wrapper &res, const fmpq_wrapper &i, unsigned long n)
+inline void mp_pow_ui(fmpq_wrapper &res, const fmpq_wrapper &i, long n)
```

Comment:
Revert this change. `mp_pow_ui` should take in a `ui`, i.e. `unsigned long` type